Mathlib/Computability/TMToPartrec.lean

```diff
@@ -649,15 +649,15 @@ theorem cont_eval_fix {f k v} (fok : Code.Ok f) :
       obtain ⟨v'', h₁, h₂⟩ := this
       rw [reaches_eval] at h₂
       swap
-      exact ReflTransGen.single rfl
+      · exact ReflTransGen.single rfl
```

Comment:
Could be `on_goal 2 => exact ReflTransGen.single rfl` removing the previous `swap`.

Reply:
I tried to recycle as much as possible the original syntax.  I would prefer this change to be a separate PR.